apps/game_backend/lib/game_backend/configuration.ex

```diff
@@ -541,4 +543,50 @@ defmodule GameBackend.Configuration do
   def get_latest_game_configuration do
     Repo.one(from(g in GameConfiguration, order_by: [desc: g.inserted_at], limit: 1))
   end
+
+  @doc """
+  Returns a 16 base encoded string representing the hashed value of the map configurations.
+  ## Examples
+      iex> get_configuration_hash_version()
+      "A6F0CC6917D195AB8A03129ACBE1FA48364845B8"
+  """
+  def get_configuration_hash_version do
```

Comment:
Why not `:erlang.term_to_binary/1` the map configuration (or whatever subfields we actually want)?

Reply:
Because the query order of everything changes the hash everytime, and we only want to change the hash when the desired values change.
We can discuss a better approach in Slack.